assets/status-portal.css

```
@@ -1,80 +1,366 @@
+  padding: 7px 10px;
+  border: 1px solid var(--border);
+  border-radius: 999px;
+  background: rgba(255, 255, 255, .025);
+  font-size: 10px;
+  letter-spacing: .14em;
+}
+.live-indicator { width: 7px; height: 7px; box-shadow: none; }
+body[data-overall-status="operational"] .live-indicator,
+body[data-overall-status="degraded"] .live-indicator,
+body[data-overall-status="maintenance"] .live-indicator,
+body[data-overall-status="outage"] .live-indicator { box-shadow: 0 0 14px currentColor; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)